war/src/main/webapp/css/style.css

```diff
@@ -1579,20 +1579,6 @@ TEXTAREA.rich-editor {
     color: #888a85;
 }
 
-#plugins .compatWarning {
```

Comment:
Have you checked they are not used anywhere else? It might make sense to just keep them as deprecated. 
CC @varyvol @batmat @raul-arabaolaza since some proprietary components might depend on that, not checked

Reply:
Yes, I searched through the repository and could not find them used anywhere else. 

How would I mark them as deprecated? Just with a CSS comment or in any special way?

Reply:
Let's wait for confirmation from CCed users. If they confirm there is no external dependency, IMHO we are good. It is highly unlikely that there are other companies which might be affected, and having a single alarm type is net positive IMHO

Reply:
Regarding marking as deprecated, yes we can just put a CSS comment. It is better to add additional comment for removing it, e.g. `TODO(andipabst): Not used after https://github.com/jenkinsci/jenkins/pull/4299, remove once there is no dependencies`

Reply:
I'm afraid we are using this internally.

Reply:
@andipabst Then we just revert it and "deprecate" in Comments, I'd guess